# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
# Commit: 970ed33359b5
# Full Hash: 970ed33359b5223277984f77c521faaa0079aa88
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Implement MediaConduitControl for controlling conduits async through state mirroring. r=ng,bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112868
# ==============================================================================

diff -r ed2e505b5873 -r 970ed33359b5 media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
--- a/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Wed Apr 21 01:13:43 2021 +0200
+++ b/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Tue Aug 10 14:13:49 2021 +0200
@@ -380,7 +380,8 @@
 
     RefPtr<MediaPipelineTransmit> audio_pipeline = new MediaPipelineTransmit(
         test_pc, transport_, GetMainThreadSerialEventTarget(),
-        test_utils->sts_target(), false, audio_conduit_);
+        AbstractThread::MainThread(), test_utils->sts_target(), false,
+        audio_conduit_);
 
     audio_pipeline->SetSendTrackOverride(audio_track_);
     audio_pipeline->Start();
@@ -414,7 +415,7 @@
 
     audio_pipeline_ = new MediaPipelineReceiveAudio(
         test_pc, transport_, GetMainThreadSerialEventTarget(),
-        test_utils->sts_target(),
+        AbstractThread::MainThread(), test_utils->sts_target(),
         static_cast<AudioSessionConduit*>(audio_conduit_.get()), nullptr,
         PRINCIPAL_HANDLE_NONE);
 
